libr/core/project.c

```diff
@@ -697,6 +697,9 @@ R_API bool r_core_project_save(RCore *core, const char *prj_name) {
 	}
 	free (script_path);
 	r_config_set (core->config, "prj.name", prj_name);
+	core->config->is_dirty = false;
```

Comment:
dont access fields directly, just use the api. `r_config_is_dirty()` will set that to false.

also, as long as we have the same logic in 3 modules, mabye good for techdebt, to have this api in `r_util` and just use that on each module like this:

`r_dirty_check()` in `libr/util/dirty.c` and expose an RDirty struct that holds the dirty boolean value and call it like this: `if (r_dirty_check (core->config->dirty)) {`  .. but if its just a boolean, maybe we dont need to hide this logic on such abstraction. what do you think?